src/gflags.cc

```diff
@@ -1163,8 +1189,8 @@ string CommandLineFlagParser::ProcessFromenvLocked(const string& flagval,
     }
 
     const string envname = string("FLAGS_") + string(flagname);
-	string envval;
-	if (!SafeGetEnv(envname.c_str(), envval)) {
+  string envval;
```

Comment:
There something wrong with indentation indeed.
My Sublime Text should use spaces for tabs, will look at this and fix.